warehouse/admin/templates/admin/malware/verdicts/index.html

```diff
@@ -0,0 +1,97 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+{% extends "admin/base.html" %}
+
+{% import "admin/utils/pagination.html" as pagination %}
+
+{% block title %}Malware Verdicts{% endblock %}
+
+{% block breadcrumb %}
+  <li class="active">Verdicts</li>
+{% endblock %}
+
+{% block content %}
+<div class="box box-primary">
+  <div class="box-body table-responsive no-padding">
+    <table class="table table-hover">
+      <tr>
+        <th>Object</th>
+        <th>Check</th>
+        <th>Classification</th>
+        <th>Confidence</th>
+        <th>Detail</th>
+      </tr>
+      {% for verdict in verdicts %}
+      <tr>
+	<td>
+	{% if verdict.project %}
+	  <a href="{{ request.route_path('admin.project.detail', project_name=verdict.project.normalized_name) }}"><i class="fa fa-cube"></i> {{ verdict.project.name }} </a>
+	{% elif verdict.release %}
+	<a href="{{ request.route_path('admin.project.release', project_name=verdict.release.project.normalized_name, version=verdict.release.version) }}"><i class="far fa-folder"></i> {{ verdict.release.project.name }}-{{ verdict.release.version }} </a>
+	{% else %}
+	  <a href="{{ request.route_path('admin.project.release', project_name=verdict.release_file.release.project.normalized_name, version=verdict.release_file.release.version) }}"><i class="far fa-file"></i> {{ verdict.release_file.filename}} </a>
+	{% endif %}
+	</td>
+	<td>
+          <a href="{{ request.route_path('admin.checks.detail', check_name=verdict.check.name) }}">
+	    {{ verdict.check.name }} v{{ verdict.check.version }}
+	  </a>
+	</td>
+        <td>
```

Comment:
might be helpful to wrap the contents of this cell in a `span` with a `title` containing `verdict.classification.value`, just to make discoverability a bit easier.

Reply:
something like

```html
          <span title="{{ verdict.classification.value }}">
            <i class="fa fa-exclamation"></i>
           {% if verdict.classification.value == 'indeterminate' %}
            <i class="fa fa-exclamation"></i>
           {% elif verdict.classification.value == 'threat' %}
            <i class="fa fa-exclamation"></i>
            <i class="fa fa-exclamation"></i>
           {% endif %}
          </span>
```